docs/shows/tokens.md

```diff
@@ -227,6 +227,56 @@ show_player:
         color2: blue
 ```
 
+## Using mpf variable values in tokens
+In the above explainations the token values, e.g. led_02 were objects you have defined in your config file. You can as well use values of variables mpf is having, e.g. machine variables, or parameter values of an event.
+
+### Event parameter(argument) values
+
+Let's assume the following event is posted
+
+```
+INFO : EventManager : Event: ======'player_turn_started'====== Args={'player': <Player 1>, 'number': 1}
+```
+The event `player_turn_started` has for example the argument `number` for the number of the player whose turn has started.
+
+``` mpf-config
+show_player:
+  player_turn_started:
+    player_num: #The name of the show to be started upon this event
+      show_tokens:
+        txt: (number)
+```
+
+The number of the player whose turn started is being displayed in the show where you have used the placeholder `txt`. The argument `number` is probably used in many events, when refering to `number` you always refer to the event which is triggering this show. You don't have to specify the event name in some kind of dot notation.
```

Comment:
Since we haven't gotten to the section on dot notation yet, it might be clearer to say something like "When using event arguments as tokens, just use the argument name without any prefix or dot notation."

Reply:
Adapted.